lib/iex/test/test_helper.exs

```diff
@@ -1,6 +1,10 @@
+assert_timeout = String.to_integer(
+  System.get_env("ELIXIR_ASSERT_TIMEOUT") || "500"
```

Comment:
I wrapped because the line below in `test/elixir/test_helper.exs` was wrapped on the same length and then I copy pasted to this file.